agent/stats/utils_unix_test.go

```diff
@@ -25,17 +25,6 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func TestDockerStatsToContainerStatsEmptyCpuUsageGeneratesError(t *testing.T) {
```

Comment:
Is this test not needed now?

Reply:
I have moved this check out of utils/unix.go to container.go

Reply:
ok, are you also moving the test to container_test.go ?

Reply:
moved the check back to this as a separate function and added the test back.